db-client-java/src/main/java/com/eventstore/dbclient/SubscriptionFilter.java

```diff
@@ -86,5 +87,56 @@ void addToWireReadReq(StreamsOuterClass.ReadReq.Options.Builder builder) {
 
         builder.setFilter(optsB.build());
     }
+
+    void addToWirePersistentReadReq(Persistent.CreateReq.AllOptions.Builder builder) {
```

Comment:
**SUGGESTION:** It might be worth aligning at least part of the logic with the regular subscription processing, as it's almost the same.